### PR TITLE
fix(console): update content of assigning new roles on app details page

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

update content of assigning new roles on app details page on target box.

## Testing

Tested locally.
image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx [hunk 1]
```diff
@@ -8,19 +8,27 @@ import * as styles from './index.module.scss';
...
         <div className={styles.name}>{name}</div>
         <div className={styles.count}>
-          ({t('user_details.roles.assigned_user_count', { value: usersCount })})
+          (
+          {t(
+            type === RoleType.User
+              ? 'user_details.roles.assigned_user_count'
+              : 'application_details.roles.assigned_app_count',
+            { value: type === RoleType.User ? usersCount : applicationsCount }
+          )}
```
Comment (charIeszhao) [inline packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx:30]: nit
```ts
type === RoleType.User ? t('user_details.roles.assigned_user_count', { value: usersCount }) : t('application_details.roles.assigned_app_count', { value: applicationsCount })
```

File: packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx [hunk 2]
```diff
@@ -8,19 +8,27 @@ import * as styles from './index.module.scss';
...
   role: RoleResponse;
+  type: RoleType;
   onDelete: () => void;
 };
 
-function TargetRoleItem({ role, onDelete }: Props) {
+function TargetRoleItem({ role, type, onDelete }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
-  const { name, usersCount } = role;
+  const { name, usersCount, applicationsCount } = role;
```
Comment (charIeszhao) [inline packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx:17]: Can you get `type` from `role`? If yes, then you don't have to pass `type` through params

**Review by charIeszhao — APPROVED** [review]
Overall looks OK

File: packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx (hunk 2 above)
Comment (darcyYe) [reply packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx:17]: Good call

Comment (darcyYe) [reply packages/console/src/components/RolesTransfer/components/TargetRoleItem/index.tsx]: Updated.

**Pull request merged**